pyproject.toml

```diff
@@ -2,14 +2,136 @@
 requires = ["setuptools", "wheel", "pybind11", "setuptools-rust"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools]
+package-dir = {"" = "src"}
+include-package-data = true
+zip-safe = false
+
+[tool.setuptools.packages.find]
+where = ["src"]
+include = ["datumaro*"]
+
+[tool.setuptools.dynamic]
+version = {attr = "datumaro.version.__version__"}
+
+[tool.pybind11.ext_modules]
+"datumaro._capi" = { sources = ["src/datumaro/capi/pybind.cpp"], extra_compile_args = ["-O3"] }
+
+[[tool.setuptools-rust.ext-modules]]
+target = "datumaro.rust_api"
+path = "rust/Cargo.toml"
+binding = "PyO3"
+
+[project]
+name = "datumaro"
+dynamic = ["version"]
+description = "Dataset Management Framework (Datumaro)"
+authors = [
+    {name = "Intel", email = "emily.chun@intel.com"}
+]
+license = {text = "MIT"}
+readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "attrs>=21.3.0",
+    "defusedxml>=0.7.0",
+    "imagesize>=1.4.1",
+    "lxml<7,>=5.2.0",
+    "numpy>=1.23.4,<2.2.0",
+    "orjson>=3.10.0",
+    "Pillow>=10.3.0",
+    "ruamel.yaml>=0.17.0",
+    "shapely>=1.7",
+    "typing_extensions>=3.7.4.3",
+    "tqdm>=4.67.1", # version with the latest security fixes
```

Comment:
Can you remove this comment please? It doesn’t make sense. If we start put it for tqdm, then to be consistent, we would have to add it to all dependencies.

Reply:
I agree, these comments are not helpful